models/members.js

```diff
@@ -13,21 +13,27 @@ const userModel = firestore.collection('users')
 
 const fetchMembers = async () => {
```

Comment:
Should this be called `fetchUsers` now? Given that all members are users, but all users are not members